src/main/java/com/fauna/client/FaunaConfig.java

```diff
@@ -100,6 +110,7 @@ public static class Builder {
         private int maxContentionRetries = 3;
         private Duration clientTimeoutBuffer = Duration.ofSeconds(5);
         private Handler logHandler = defaultLogHandler();
+        private StatsCollector statsCollector;
```

Comment:
I see you're not setting one by default. Should we document performance considerations? Otherwise it'd clean up the code to always set the default one.

Reply:
Hmm, I chose not to set one because I was worried that users needed to read and reset. But seems that's not true. I just read the docs and overflow/underflow just roll over so we don't care.

I'll simplify